src/models/form.ts

```diff
@@ -0,0 +1,18 @@
+import { Prop, PropType, Ref } from "@typegoose/typegoose";
+import SignatureChainLink from "./signature-chain-link";
+import FormInstance from "./form-instance";
+
+class Form {
+    //! Still confused on how IDs work here???? Do I need to add one? 😭
+    
+    @Prop({ required: true, type: () => String }) //? Also be unique?
+    public name!: string;
+
+    @Prop({ required: true, ref: () => SignatureChainLink, type: () => SignatureChainLink })
+    public signatureChainLinkHead!: Ref<SignatureChainLink>;
+
+    @Prop({ required: true, ref: () => FormInstance, type: () => [FormInstance] }, PropType.ARRAY)
+    public formInstances!: Ref<FormInstance>[];
```

Comment:
I don't think this field is necessary either - I think we just had it bc of weird TypeORM stuff

Reply:
Yeah I don't think so either? But who knows might be a nice convenience field?